backend/migrations/20240214192738_food_env_score.up.sql

```diff
@@ -1,17 +1,16 @@
 -- Add up migration script here
 
-CREATE DOMAIN env_rating as smallint CHECK (VALUE > 0);
```

Comment:
does this work with `INT`? In postgres it is at least convention to use lowercase for types (`integer`)

Reply:
In other tables we wrote that out to `integer` nor just `int` so I would prefer that for consistancy